src/application/router/routes.ts

```diff
@@ -4,6 +4,9 @@ import Note from '@/presentation/pages/Note.vue';
 const routes = [
   {
     path: '/',
+    meta: {
+      layout: 'DefaultLayout',
```

Comment:
could this property be optional so we won't include it if the Default layout is used